tests/testthat/helpers-with-settings.R

```diff
@@ -30,6 +30,31 @@ app_tws <- function() {
   )
 }
 
+#' Verbatim popup app
+#'
+#' @description Example table with setting app for testing using \code{shinytest2}
+#'
+#' @keywords internal
+#'
+app_vpu <- function() {
+  shiny::shinyApp(
+    ui = shiny::fluidPage(
+      verbatim_popup_ui(
+        id = "verbatim_popup",
+        button_label = "Open me"
+      )
+    ),
+    server = function(input, output, session) {
+      verbatim_popup_srv(
+        id = "verbatim_popup",
+        "if (TRUE) { print('Popups are the best') }",
+        title = "Verbatim popup title",
```

Comment:
should custom text like `title` or `button_label` be parametrized and be passed as arguments of `app_vpu` function?